.gitignore

```diff
@@ -21,7 +21,7 @@ target
 /esvm
 .htpasswd
 .eslintcache
-plugins
+/plugins
```

Comment:
I'm all for tightening up this file. I'm guessing we're ignoring more that we intend here. I don't think this PR is the right place to do it, and that it deserves it's own. I modified the `plugin` ignore setting because it directly affects this PR.